### PR TITLE
Additional provision script for transactor clients

### PR DESCRIPTION
**Pull request by vyzo**
Like transactor, but also pulls medichain-client and configures /mnt/data instead of /mnt/transactor.

### CODE REVIEW

File: provisioning/playbooks/transactor-client-provisioning.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- hosts: transactor-client-provisioning
+  roles:
+    - transactor
+
+  tasks:
+  - name: install pip
+    apt: name=python-pip state=present
+
+  - name: update to latest pip
```
Comment (parkan) [inline provisioning/playbooks/transactor-client-provisioning.yml:10]: so this internally runs `pip install -U pip`?

Comment (parkan) [inline provisioning/playbooks/transactor-client-provisioning.yml:10]: ah yeah I see, you're using `pip` to install `pip` to `latest`, makes sense

File: provisioning/playbooks/transactor-client-provisioning.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    pip: name=pip state=latest
+    
+  - name: checkout mediachain
```
Comment (bigs) [inline provisioning/playbooks/transactor-client-provisioning.yml:13]: maybe add a variable `mediachain_branch` that defaults to `master` here

**Comment by bigs** [comment]
i kno it's a bit of a pain (& would be happy to help) but it would nice to have this in a role so we can reuse it/compose it. looks good

**Comment by vyzo** [comment]
I've been thinking of refactoring both provisioning scripts to extract some roles, so will do.

**Comment by vyzo** [comment]
I think I'll leave the role refactoring for next pass; the pain level exceeds my current tolerance :)

**Pull request merged**